try-it-out/docs/pages/deploy-devops-tools.rst

```diff
@@ -0,0 +1,63 @@
+.. _Deploy Devops Tools:
+-----------------------
+Deploy The Devops Tools
+-----------------------
+
+In this section, you will be deploying all of the DevOps tools that we will be using within this tutorial. This consists of deploying a Jenkins instance, a SonarQube server, and a local (insecure) Docker registry. 
+
+.. note:: The tools that we will be utilizing throughout this tutorial are not fully secure and thus not production-ready. They are meant to just be a showcase of what SDP itself can do. 
+
+To begin, clone our SDP-Labs_ GitHub repository, which includes all of the files you'll need to use throughout our guide. 
+
+.. _SDP-Labs: https://github.com/boozallen/sdp-labs
+
+.. code-block:: bash
+
+   ## cloning via ssh
+   git clone git@github.com:boozallen/sdp-labs.git
+   ## cloning via https
+   git clone https://github.com/boozallen/sdp-labs.git
+
+**If you haven't already**, use one of the above commands to clone the repository to your computer. 
```

Comment:
I know we used the "**If you haven't already**" disclaimer in previous docs b/c we had the instruction replicated in the docs. Since that's no longer true, is it reasonable to say this can be inferred?